doc/reference/actions/format-dune-file.rst

```diff
@@ -0,0 +1,13 @@
+cat
+---
+
+.. highlight:: dune
+
+.. describe:: (format-dune-file <file>)
+
+   Print the formatted contents of a file, assumed to contain S-expressions, to
+   stdout.
```

Comment:
Shouldn't it emphasize the `dune` version used to format? What do you think about adding a reference to `dune format-dune-file` so the information can be linked?

Reply:
You are right, it is better to not mentioning it. The documentation looks good to me :+1: